src/components/catalog/catalog.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { Filter } from './filter';
+import { PostersOverview } from './posters-overview';
+import { Sorting } from './sorting';
+import { FilterSortingWrapper, NumberMovies } from './styles';
+
+export const Catalog: FC = () => (
+  <>
+    <FilterSortingWrapper>
```

Comment:
I think must be a better name for this component

Reply:
Ok, I think of this point.